Changelog Apr 17 (#464)
CloudCannon/platform-documentation: changelogs/2025-04-17_cloudcannon-open-beta-publishing-conflict-resolution.mdx

---
_schema: default
title: 'CloudCannon Open Beta: Publishing Conflict Resolution'
---
Our Beta is now available to everyone!

The Publishing Conflict Resolution Beta introduces Publishing Conflict resolution in app, appearance and functionality improvements to the *Publishing* page, *Status* page, and *Domains* section, an updated *Source Editor*, and text diffs for unsaved changes in our editing interfaces.

We'd like to invite everyone to test our new features before the full launch. Taking part in this Beta is a great chance to let us know what’s working, and what could be improved for you and your team.

> For more information about the features included in this Beta, please read our [blog post](/blog/open-beta-publishing-conflict-resolution/) or check out the [Beta Changelog](https://cloudcannon.com/documentation/articles/beta-changelog/).

Reviewer: ```suggestion
For more information about the features included in this Beta, please read our [blog post](/blog/open-beta-publishing-conflict-resolution/) or check out the [Beta Changelog](/documentation/articles/beta-changelog/).
```

### Using CloudCannon Beta

To use the CloudCannon Beta, you must switch your account to the Beta environment.

To enter the CloudCannon Beta:

1. Log in to CloudCannon and open your *Site Dashboard*.
2. Click on your avatar in the bottom left of the app.
3. Select *Enter CloudCannon Beta* from the *Account Menu*.
4. Confirm by clicking the *Enter Beta* button on the *Try our new Beta* model.
5. CloudCannon will reload.

Alternatively, you can click the *Enter Beta* button in the *App Sidebar*.

> The Beta environment is now enabled for any Site you open in CloudCannon. To return to the Main CloudCannon environment, click *Exit CloudCannon Beta* in the *Account Menu*.

Reviewer: ```suggestion
The Beta environment is now enabled. To return to the Main CloudCannon environment, click *Exit CloudCannon Beta* in the *Account Menu*.
```

### Beta Feedback

If you have any questions or feedback regarding this Beta release, you can discuss them on our new [CloudCannon Community](https://community.cloudcannon.com/) or get in contact with our friendly [support team](https://cloudcannon.com/support/).

---

> This release improved our file server performance and reliability, and added the *Use Relative URLs when redirecting* flag, comments for DNS records, and NS DNS records for subdomains.

Reviewer: ```suggestion
This release improved our file server performance and reliability, added the *Use Relative URLs when redirecting* flag, comments for DNS records, and NS DNS records for subdomains.
```

> It also addressed an issue affecting Snippets in Rich Text.

Reviewer: ```suggestion
It also addressed an issue affecting Snippets in the Visual Editor.
```

## Features & Improvements

> * Improved file server performance and reliability for better hosting.

Reviewer: I might be wrong, but this might not be going out - might be worth double checking with George.

* Added the *Use Relative URLs when redirecting* flag under *Org Settings* and *Site Settings* to use relative redirects rather than absolute redirects.
* You can now add a comment to DNS records, allowing you to write a reminder description of the record's purpose.
* You can now add an NS record for your Custom Domain subdomains.

## Fixes

> * Fixed an issue where, in some cases, Snippets in Rich Text would break.

Reviewer: ```suggestion
* Fixed an issue where the snippet option was incorrectly included or excluded in the editor toolbar.
```
